submissions/migrations/0004_remove_django_extensions.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0003_submission_status'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='uuid',
+            field=models.UUIDField(default=uuid.uuid4, db_index=True),
+        ),
+    ]
```

Comment:
Running against a sandbox with 200K rows on the table in question:

```
edxapp@ip-192-168-0-15:~/edx-platform$ time ./manage.py lms --settings=aws migrate submissions 0004
<lots of django warnings>
2017-08-02 09:56:21,582 INFO 12833 [dd.dogapi] dog_stats_api.py:66 - Initializing dog api to use statsd: localhost, 8125
Operations to perform:
  Target specific migration: 0004_remove_django_extensions, from submissions
Running migrations:
  Rendering model states... DONE
  Applying submissions.0004_remove_django_extensions... OK

real	0m6.412s
user	0m5.656s
sys	0m0.676s
```

From my perspective, the `Applying submissions.0004_remove_django_extensions` portion was nearly instantaneous, the 5 seconds were spent in django warmups and model state rendering.